.readthedocs.yaml

```diff
@@ -0,0 +1,21 @@
+# Read the Docs configuration file for Sphinx projects
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+version: 2
+
+build:
+  os: ubuntu-22.04
+
+python: 
+  version: 3.12
+  install:
+  - requirements: manual/requirements.txt
+
+# Build documentation in the 'manual' directory with Sphinx
+sphinx:
+  configuration: manual/conf.py
+
+# Build docs in additional formats: PDF and ePub
+formats:
+   - pdf
+   - epub
```

Comment:
Currently, we don't offer those. We don't have any code to make PDF format nicely, so it's probably best if people just stick to the HTML version. They can download the HTML version for offline viewing, so let's just keep it that way.